x-pack/plugins/security_solution/public/flyout/left/components/response_details.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiSpacer, EuiTitle } from '@elastic/eui';
+import styled from 'styled-components';
+import { RESPONSE_DETAILS_TEST_ID, RESPONSE_EMPTY_TEST_ID } from './test_ids';
+import { expandDottedObject } from '../../../../common/utils/expand_dotted';
+import type {
+  ExpandedEventFieldsObject,
+  RawEventData,
+} from '../../../../common/types/response_actions';
+import { useLeftPanelContext } from '../context';
+import { useIsExperimentalFeatureEnabled } from '../../../common/hooks/use_experimental_features';
+import { useOsqueryTab } from '../../../common/components/event_details/osquery_tab';
+import { useResponseActionsView } from '../../../common/components/event_details/response_actions_view';
+import * as i18n from './translations';
+
+export const RESPONSE_TAB_ID = 'response-details';
+
+const InlineBlock = styled.div`
+  display: inline-block;
+  line-height: 1.7em;
+`;
+
+export const ResponseDetails: React.FC = () => {
```

Comment:
would you mind puttin a small description here? I know it might not bring that much value but it's for consistency with the other components/interfaces/hooks we have in the `flyout` folder